-- Fix pyproject.toml license field to conform to PEP 621.
-- Newer setuptools rejects the bare string form of project.license.
-- See: https://packaging.python.org/en/latest/specifications/declaring-project-metadata/#license

--- pyproject.toml.orig	2026-09-01 20:51:59 UTC
+++ pyproject.toml
@@ -15,8 +15,7 @@ keywords = [ "csl", "citation", "html", "rst", "bibtex
 description = "Citations and bibliography formatter"
 
 keywords = [ "csl", "citation", "html", "rst", "bibtex", "xml" ]
-license = "BSD-2-Clause-Views"
-license-files = [ "LICENSE" ]
+license = {text = "BSD-2-Clause-Views"}
 classifiers = [
     "Development Status :: 3 - Alpha",
     "Environment :: Console",
@@ -53,6 +52,7 @@ provides = [ "citeproc" ]
 
 [tool.setuptools]
 provides = [ "citeproc" ]
+license-files = [ "LICENSE" ]
 package-data.citeproc = [
             "data/locales/*.xml",
             "data/locales/locales.json",
